osu.Game.Rulesets.Osu/Objects/Drawables/DrawableSpinner.cs

```diff
@@ -331,7 +334,7 @@ private void updateBonusScore()
 
             while (completedFullSpins.Value != spins)
             {
-                var tick = ticks.FirstOrDefault(t => !t.Result.HasResult);
+                var tick = ticks.FirstOrDefault(t => !t.Result.HasResult && !(t is DrawableSpinnerHealthTick));
```

Comment:
would prefer to use `(... is not ...)` syntax rather than `!(... is ...)` (applies to every usage of this syntax)

Reply:
Should be fixed.